lib/services/ios-debug-service.ts

```diff
@@ -2,17 +2,19 @@
 "use strict";
 
 import * as iOSDevice from "../common/mobile/ios/device/ios-device";
-
 import * as net from "net";
 import * as path from "path";
 import * as semver from "semver";
 import byline = require("byline");
 
-let InspectorBackendPort = 18181;
+const InspectorBackendPort = 18181;
+const inspectorAppName = "NativeScript Inspector.app";
```

Comment:
why the first constant name `InspectorBackendPort` starts with capital letter and the others do not